cmd/cp-main.go

```diff
@@ -41,13 +41,13 @@ var (
 			Name:  "recursive, r",
 			Usage: "copy recursively",
 		},
-		cli.IntFlag{
+		cli.StringFlag{
 			Name:  "older-than",
-			Usage: "copy objects older than N days",
+			Usage: "copy objects older than N hours and M minutes",
 		},
-		cli.IntFlag{
+		cli.StringFlag{
 			Name:  "newer-than",
-			Usage: "copy objects newer than N days",
+			Usage: "copy objects newer than N hours and M minutes",
```

Comment:
Yes as @nitisht suggested, i feel it would be a bit complex for filtering days. It can be `ddhhmm` in my opinion.

Reply:
`--expire, -E "168h"          Set expiry in NN[h|m|s]. ` mc expiry uses hours even though 7 days is the default, so i came up with hours and minutes. Will incorporate the days as well.